AK/Platform.h

```diff
@@ -79,6 +79,11 @@
 #endif
 #define NO_SANITIZE_ADDRESS __attribute__((no_sanitize_address))
 
+#ifdef NO_SANITIZE_ADDRESS
+#    undef NO_SANITIZE_ADDRESS
```

Comment:
You undeff'd the wrong thing :D

we want NO_SANITIZE_ADDRESS to keep meaning something.

Reply:
That's what I get for copy-paste. Fixed :^)